keywords/LiteServiOS.py

```diff
@@ -32,8 +61,8 @@ def install(self):
         if self.storage_engine != "SQLite":
             raise LiteServError("https://github.com/couchbaselabs/liteserv-ios/issues/1")
 
-        package_name = "couchbase-lite-ios-liteserv-{}.app".format(self.version_build)
-        app_path = "{}/{}".format(BINARY_DIR, package_name)
+        package_name = "LiteServ-iOS.app"
+        app_path = "{}/{}/{}".format(BINARY_DIR, "LiteServ-iOS", package_name)
```

Comment:
Quick clarification. The new code in LiteServiOS.py was already in the branch right? The LiteServ-iOS.app currently being built is only for emulator. I and Seth had discussed adding some logic to detect the device and continue with emulator if no device is found. In that case, the host and a few other variables have to be reset. You can implement what you think is best.